hack/verify-api-groups.sh

```diff
@@ -112,8 +112,8 @@ for expected_install_package in "${expected_install_packages[@]}"; do
 	fi
 
 	for known_version_file in "${known_version_files[@]}"; do
-		if ! grep -q "${expected_install_package}/install" ${known_version_files} ; then
-			echo "missing ${expected_install_package}/install from ${known_version_files}"
+		if ! grep -q "${expected_install_package}/install" "${known_version_file}" ; then
```

Comment:
I think if you go digging the file on line 101 does not exist and we patched this out in one of the other scripts

Reply:
@liggitt might be someone to ask :^)

Reply:
@BenTheElder 
Yes, this file does not exist.
```
pkg/client/clientset_generated/internal_clientset/import_known_versions.go: No such file or directory
```
But how do we fix it? Any suggestions? @liggitt

Reply:
this was removed in https://github.com/kubernetes/kubernetes/pull/71136

we can drop the `clientset_generated/internal_clientset` line

Reply:
Thanks a lot. @liggitt 
I have updated the code.